src/packet/BinaryWriter.js

```diff
@@ -1,137 +1,62 @@
-﻿'use strict';
+'use strict';
 /*
  * Simple BinaryWriter is a minimal tool to write binary stream with unpredictable size.
  * Useful for binary serialization.
  *
- * Copyright (c) 2016 Barbosik
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *     http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- * 
+ * Copyright (c) 2016 Barbosik https://github.com/Barbosik
+ * License: Apache License, Version 2.0
  */
 
-function BinaryWriter(size) {
-    if (!size || size <= 0) {
-        size = Buffer.poolSize / 2;
-    }
-    this._buffer = new Buffer(size);
+function BinaryWriter() {
+    this._buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);

```

Comment:
> @MastaCoder You claimed you have done much. WHERES THE PROOF!!
> You have worked on:
> - Ogar
> - Cigar
> - MultiOgar
> - PolarsBots
> THATS IT!
- You think that two people can make company-grade software in 5 years? Microsoft started in a garage and needed 30 years to gain momentum with Windows 95.

> @Luka967 You have some stuff on your profile. BUT IF I TAKE A CLOSER LOOK, THEY ARE NOTHING!
> Take out all game-related stuff and you have a empty page
> You, have also been around longer than I have. And yet you show no progress as well.
> You guys, are not learning anything new, but are instead wasting your time away with a game. 
> STOP WASTING YOUR LIFE!
- **Do you think that I care what you're talking about me? I don't care that I'm wasting my life.**

> You guys, are not learning anything new, but are instead wasting your time away with a game. 
- MastaCoder learned how to handle MySQL and PHP well since he made the agar-like client.
- I learned Javascript, node.js, Python, HTML, CSS and C# since I made this profile.
- **And developing games can earn you money, and money is practically future, do you know that?**

>STOP WASTING YOUR LIFE!
- This is literally like 60,000 years ago when you invent a new writing script and all the other tribes said this, but over time a lot of them joined you.
- Why should you take a look *only* at my Github profile?